src/js/view/controls/tizen/tizen-controlbar.ts

```diff
@@ -186,8 +180,9 @@ export default class TizenControlbar extends Controlbar {
                 }
                 break;
             case 39: // right-arrow
-                if (isAdsMode && this.adSkipButton && activeButton === this.elements.play) {
-                    this.setActiveButton(this.adSkipButton);
```

Comment:
Where `adSkipButton` could be passed into setActiveButton

Reply:
Updated!